### PR TITLE
2 GN cards 1 CLB card

### LINKED ISSUE #852: Flaming Fist Officer

### PR DESCRIPTION
**Pull request by JohnWilliams77**
-closes #852

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/zamriel_seraph_of_steel.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Name:Zamriel, Seraph of Steel
+ManaCost:2 W W
+Types:Legendary Creature Angel
+K:Flying
+PT:3/4
+S:Mode$ Continuous | Affected$ Creature.YouCtrl+equipped | AddKeyword$ Indestructible | Condition$ PlayerTurn | Description$ As long as it's your turn, creatures your control have indestructible.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/zamriel_seraph_of_steel.txt:6]: Missing „equipped“

File: forge-gui/res/cardsfolder/upcoming/vogar_necropolis_tyrant.txt [hunk 2]
```diff
@@ -0,0 +1,12 @@
+Name:Vogar, Necropolis Tyrant
+ManaCost:3 B B
+Types:Legendary Creature Zombie Giant
+PT:4/4
+K:Menace
+T:Mode$ ChangesZone | ValidCard$ Creature.Other+YouCtrl | Origin$ Battlefield | Destination$ Graveyard | TriggerZones$ Battlefield | Execute$ TrigPutCounter | PlayerTurn$ True | TriggerDescription$ Whenever another creature you control dies during your turn, put a +1/+1 counter on CARDNAME.
+SVar:TrigPutCounter:DB$ PutCounter | CounterType$ P1P1 | CounterNum$ 1
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Card.Self | Execute$ TrigDraw | TriggerDescription$ When Vogar dies, draw a card for each +1+1 counter on it.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/vogar_necropolis_tyrant.txt:8]: Use NICKNAME

File: forge-gui/res/cardsfolder/upcoming/flaming_fist_officer.txt [hunk 3]
```diff
@@ -0,0 +1,7 @@
+Name:Flaming Fist Officer
+ManaCost:2 W
+Types:Creature Gnome Soldier
+PT:2/2
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Any | ValidCard$ Creature.Other+YouCtrl | TriggerZones$ Battlefield | Execute$ TrigPutCounter | TriggerDescription$ Whenever another creature you control leaves the battlefield, put a +1/+1 counter on CARDNAME.
+SVar:TrigPutCounter:DB$ PutCounter | Defined$ Self | CounterType$ P1P1 | CounterNum$ 1
+Oracle:Whenever another creature you control leaves the battlefield, put a +1/+1 counter on Flaming Fist Officer.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/flaming_fist_officer.txt:7]: DeckHas:Ability$Counters

File: forge-gui/res/cardsfolder/upcoming/zamriel_seraph_of_steel.txt [hunk 4]
```diff
@@ -0,0 +1,8 @@
+Name:Zamriel, Seraph of Steel
+ManaCost:2 W W
+Types:Legendary Creature Angel
+K:Flying
+PT:3/4
+S:Mode$ Continuous | Affected$ Creature.YouCtrl+equipped | AddKeyword$ Indestructible | Condition$ PlayerTurn | Description$ As long as it's your turn, equipped creatures your control have indestructible.
+DeckHints:Type$Artifact|Equipment
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/zamriel_seraph_of_steel.txt:7]: Probably just `Type$Equipment` would be better here

File: forge-gui/res/cardsfolder/upcoming/zamriel_seraph_of_steel.txt [hunk 5]
```diff
@@ -0,0 +1,8 @@
+Name:Zamriel, Seraph of Steel
+ManaCost:2 W W
+Types:Legendary Creature Angel
+K:Flying
+PT:3/4
+S:Mode$ Continuous | Affected$ Creature.YouCtrl+equipped | AddKeyword$ Indestructible | Condition$ PlayerTurn | Description$ As long as it's your turn, equipped creatures your control have indestructible.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/zamriel_seraph_of_steel.txt:6]: `your control` -> `you control`

**Review by Northmoc — COMMENTED** [review]
Nice! Just a few little things to tweak

Thank you!

**Review by Northmoc — APPROVED** [review]
Looks great! Thanks again,

**Pull request merged**